includes/drush/custom/toolkit.drush.inc

```diff
@@ -69,9 +69,8 @@ function drush_toolkit_check_modules_authorized_security($project_id = NULL) {
     // Check update module status.
     $updateStatus = system_get_info('module', 'update');
```

Comment:
I dont' think we need anymore this variable after the change bellow.

Reply:
Also removed.